assets/templates/partials/census/related-content.tmpl

```diff
@@ -0,0 +1,12 @@
+<section id="related-content" aria-label="{{ localise "RelatedContentTitle" .Language 1 }}">
+    <h2 class="ons-u-mt-xl ons-u-pb-no ons-u-pt-no">{{ localise "RelatedContentTitle" .Language 1 }}</h2>
+    {{ range $i, $item := .DatasetLandingPage.RelatedContentItems }}
```

Comment:
You've assigned the index var `$i` but not used it. Could just do a (simpler) range 
```suggestion
    {{ range .DatasetLandingPage.RelatedContentItems }}
```

Reply:
Thanks for suggestion - I've updated